pyaedt/modules/PostProcessor.py

```diff
@@ -2392,7 +2386,7 @@ def get_scalar_field_value(
             Dictionary of all variation variables with their values.
             e.g. ``['power_block:=', ['0.6W'], 'power_source:=', ['0.15W']]``
             The default is ``None``.
-        isvector : bool, optional
+        is_vector : bool, optional
             Whether the quantity is a vector. The  default is ``False``.
         intrinsics : str, optional
             This parameter is mandatory for a frequency field
```

Comment:
```suggestion
            This parameter is mandatory for a frequency field.
```
This is no brief description. Please add one.